appendices/extension_index.txt

```diff
@@ -53,7 +53,7 @@
 * <<object_types_sampler_image3D, `KHR_SAMPLER_IMAGE3D`>>: `image3D` sampler subtype is supported
 * <<object_types_sampler_primitive, `KHR_SAMPLER_PRIMITIVE`>>: `primitive` sampler subtype is supported
 * <<object_types_sampler_transform, `KHR_SAMPLER_TRANSFORM`>>: `transform` sampler subtype is supported
-* <<object_types_spatial_field, `KHR_SPATIAL_FIELD_FILTER_CUBIC`>>: spatial fields with parameter `filter` support `cubic` filtering
+* <<object_types_spatial_field, `KHR_SPATIAL_FIELD_STRUCTURED_REGULAR_FILTER_CUBIC`>>: structured regular spatial fields with parameter `filter` support `cubic` filtering
```

Comment:
the "with parameter `filter`" was to exclude e.g. Unstructured; now StructuredRegular *has* param `filter`; also, the link anchor needs to change as well; and it should move 1 line down, after the StructuredRegular extension
like
> * <<object_types_spatial_field_structured_regular, `KHR_SPATIAL_FIELD_STRUCTURED_REGULAR_FILTER_CUBIC`>>: `structuredRegular` spatial field supports `cubic` filtering for parameter `filter`